### PR TITLE
[SDESK-875] - Allow duplication to different desk/stage

### PR DESCRIPTION
**Pull request by akintolga**
Depends on https://github.com/superdesk/superdesk-core/pull/838

### CODE REVIEW

File: scripts/apps/archive/index.js [hunk 1]
```diff
@@ -155,6 +155,26 @@ angular.module('superdesk.apps.archive', [
...
+                icon: 'copy',
+                monitor: true,
+                controller: ['data', 'send', function(data, send) {
+                    send.allAs([data.item], 'duplicateTo');
+                }],
+                filters: [
+                    {action: 'list', type: 'archive'},
+                    {action: 'list', type: 'archived'}
+                ],
+                keyboardShortcut: 'ctrl+alt+t',
```
Comment (sjunaid) [inline scripts/apps/archive/index.js]: LGTM, only one minor thing that, `ctrl+alt+t` interfering and opening terminal window when FF/Chrome running under ubuntu(just an alt. suggestion if shortcut is required: ctrl+alt+c / ctrl+alt+r), also alt+k shortcut list need to be update for `Monitoring / List actions` section

Comment (akintolga) [inline scripts/apps/archive/index.js]: Thanks @sjunaid I'll change it to ctrl+alt+r

**Comment by ioanpocol** [comment]
Please rebase in order to pass the tests

**Comment by akintolga** [comment]
Rebased, one more test failing, working on that.

**Pull request merged**